### PR TITLE
Added Agricultural Expansion

### PR DESCRIPTION
**Pull request by zackeezy**

### CODE REVIEW

**Comment by RlonRyan** [comment]
I'll have to have someone go through this and verify all the plants, last time I merged a large plant set without careful inspection there turned out to be a lot of bugs. Hopefully we can merge this soon though. Thanks for your contribution!

**Comment by zackeezy** [comment]
I understand.  I feel like there's probably something I missed, but it was just a matter of ctrl-f the current resource, and find and replace with the new resource.  I've been using it for a while so it seems pretty good.

File: mod_agriculturalexpansion/plants/apatite_sprout_plant.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    "seed_texture": "agriculturalexpansion:items/apatite_sprout",
+    "plant_textures": [
+      "agriculturalexpansion:blocks/crop0",
+      "agriculturalexpansion:blocks/crop1",
+      "agriculturalexpansion:blocks/crop2",
+      "agriculturalexpansion:blocks/crop3",
+      "agriculturalexpansion:blocks/crop4",
+      "agriculturalexpansion:blocks/crop5",
+      "agriculturalexpansion:blocks/apatite_crop6",
+      "agriculturalexpansion:blocks/apatite_crop7"
```
Comment (RlonRyan) [inline mod_agriculturalexpansion/plants/apatite_sprout_plant.json]: Why are the last two textures different?

Comment (TokkenGmr) [inline mod_agriculturalexpansion/plants/apatite_sprout_plant.json]: Looks like the first 5 stages are generic textures that are used for all the crops and the last 2 refer to the actual "resource".

Comment (zackeezy) [inline mod_agriculturalexpansion/plants/apatite_sprout_plant.json]: That's what the textures are called in the assets.  The first five progressive textures are standard across all crops in the mod, but the last two are resource specific.

Comment (RlonRyan) [inline mod_agriculturalexpansion/plants/apatite_sprout_plant.json]: I see... that's a novel way of reusing textures.

**Comment by RlonRyan** [comment]
Looks good to me. Might as well merge it now! Thank you very much for your contribution!

**Pull request merged**